tokens/governance/treasury-spend.md

```diff
@@ -0,0 +1,93 @@
+ - **Minimum Bond** — the minimum bond accepted with a proposed treasury spend regardless of the amount of the treasury spend. This minimum bond amount may render small treasury proposal amounts infeasible
+ - **Treasury address** — the address where treasury funds accrue and are disbursed from
+ - **Beneficiary** — the address, such as a [Moonbeam Safe multisig](/tokens/manage/multisig-safe/){target=_blank}, that will receive the funds of the treasury proposal if enacted
+ - **Value** — the amount that is being asked for and will be allocated to the beneficiary address if the treasury proposal is enacted
+ - **Spend period** — The waiting period after a treasury proposal has been approved but before the funds have been disbursed to the beneficiary
+
+=== "Moonbeam"
+    |         Variable         |                                                          Value                                                          |
+    |:------------------------:|:-----------------------------------------------------------------------------------------------------------------------:|
+    |      Proposal Bond       | {{ networks.moonbeam.treasury.proposal_bond}}% of proposed spend  |
+    |     Minimum Bond     |   {{ networks.moonbeam.treasury.proposal_bond_min}} GLMR |
+    |     Spend Period     |   {{ networks.moonbeam.treasury.spend_period_days}} days |
+    |     Treasury Address     |   [0x6d6f646C70792f74727372790000000000000000](https://moonscan.io/address/0x6d6f646C70792f74727372790000000000000000){target=_blank} |
+
+=== "Moonriver"
+    |         Variable         |                                                           Value                                                           |
+    |:------------------------:|:-------------------------------------------------------------------------------------------------------------------------:|
+    |      Proposal Bond       | {{ networks.moonriver.treasury.proposal_bond}}% of proposed spend  |
+    |     Minimum Bond     |   {{ networks.moonriver.treasury.proposal_bond_min}} MOVR |
+    |     Spend Period     |   {{ networks.moonriver.treasury.spend_period_days}} days |
+    |     Treasury Address     |   [0x6d6f646C70792f74727372790000000000000000](https://moonriver.moonscan.io/address/0x6d6f646C70792f74727372790000000000000000){target=_blank} |
+    
+=== "Moonbase Alpha"
+    |         Variable         |                                                          Value                                                          |
+    |:------------------------:|:-----------------------------------------------------------------------------------------------------------------------:|
+    |      Proposal Bond       | {{ networks.moonbase.treasury.proposal_bond}}%  |
+    |     Minimum Bond     |   {{ networks.moonbase.treasury.proposal_bond_min}} DEV |
+    |     Spend Period     |   {{ networks.moonbase.treasury.spend_period_days}} days |
```

Comment:
It doesn't show anything on Moonscan for Moonbase in particular, but maybe we should use Subscan links since the Treasury is a Substrate feature 
https://moonbase.subscan.io/account/0x6d6F646c70632f74727372790000000000000000